### ISSUE TITLE
Reproducing the Feed-Forward 3D Gaussian Splattings  visual results

### ISSUE THREAD

**Issue by gardiens** [ByteDance-Seed/Depth-Anything-3#44]
Hello,
Thank you for sharing these impressive results.
I am currently attempting to reproduce the high-quality 3D Gaussian splatting outputs shown in your work. However, I could not find a notebook or detailed instructions describing how these results were generated, and my own attempts currently produce unexpected artifacts.

Below is the minimal reproducible example I used, along with the corresponding depth output:
```
import os
import numpy as np
import matplotlib.pyplot as plt
from PIL import Image
import torch
from depth_anything_3.api import DepthAnything3
from depth_anything_3.utils.visualize import visualize_depth
device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
model = DepthAnything3.from_pretrained("depth-anything/DA3-SMALL")
model = model.to(device)
model.eval()
print(f"Model loaded on {device}")
image_base_path="/workspaces/Depth-Anything-3/data/DL3DV-10K/1K/83d6ec4ff945f82af40feea80b4cdbdef26bbfdfd8792e05b733cde838d5bd27/images_8"
# list all png files in the directory
image_paths = [os.path.join(image_base_path, f) for f in os.listdir(image_base_path) if f.endswith('.png')]
len(image_paths)
image_paths=image_paths[:50]
# Export multiple formats including Gaussian Splatting
# Note: infer_gs=True requires da3-giant or da3nested-giant-large model
model = DepthAnything3(model_name="da3nested-giant-large").to("cuda")

prediction = model.inference(
    image=image_paths,
    extrinsics=None,
    intrinsics=None,
    export_dir="./output",
    export_format="npz-glb-gs_ply-gs_video",
    align_to_input_ext_scale=True,
    infer_gs=True,  # Required for gs_ply and gs_video exports
)

fig, axes = plt.subplots(2, n_images, figsize=(12, 6))

if n_images == 1:
    axes = axes.reshape(2, 1)

for i in range(n_images):
    # Show original image
    if prediction.processed_images is not None:
        axes[0, i].imshow(prediction.processed_images[i])
    axes[0, i].set_title(f"Input {i+1}")
    axes[0, i].axis('off')
    
    # Show depth map
    depth_vis = visualize_depth(prediction.depth[i], cmap="Spectral")
    axes[1, i].imshow(depth_vis)
    axes[1, i].set_title(f"Depth {i+1}")
    axes[1, i].axis('off')

plt.tight_layout()
plt.show()
```

<img width="1189" height="528" alt="Image" src="https://github.com/user-attachments/assets/615a844e-6e81-4a98-a240-70a5dab06d7f" />

**Comment by etola** [comment]
Can you try initializing the da3nested-giant-large model you use like this. I think it is not downloading the weights the way you are doing it.


```
 model_dir = "depth-anything/DA3NESTED-GIANT-LARGE"
 model = DepthAnything3.from_pretrained(model_dir)
 model = model.to("cuda")
```

**Comment by gardiens** [comment]
Thank you for the catch ! 
It works indeed 
<img width="1189" height="528" alt="Image" src="https://github.com/user-attachments/assets/a6ec4640-e2f0-4ac6-ab1a-111eb2ee98b8" />

**Comment by ZhaoLizz** [comment]
You are using the DA3-SMALL model, but only DA3NESTED-GIANT-LARGE and DA3-GIANT support Guassian Splatting, according to the authors' README.
